lib/src/mock_canvas.dart

```diff
@@ -78,7 +82,7 @@ class MockCanvas extends Fake implements Canvas, Matcher {
 
   @override
   Description describe(Description description) {
-    description.add('Canvas$_commands');
+    description.add('Canvas {\n${_commands.join('\n')}\n}');
```

Comment:
I just found that its much easier to read like this when there are a handful of commands